source/fundamentals/logging.txt

```diff
@@ -0,0 +1,285 @@
+=======
+Logging
+=======
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. _logging:
+
+Overview
+--------
+
+.. What is on this page?
+
+In this guide, we show you how to set up and configure a logger in the
+MongoDB Java driver.
+
+.. What do any new terms mean?
+
+.. Not applicable. We are treating definition of logger as understood.
+
+.. What can you expect to see on this page?
+
+You will learn how to:
+
+- Set up a logger using the Simple Logging Facade For Java (SLF4J)
+- Configure the log level of your logger
+
+.. Who should read this? Where should I go if this isn't the page I was looking for?
+
+.. Not Applicable. No natural redirect location.
+
+Set Up a Logger
+---------------
+
+In this section, we provide background on the dependencies necessary to set up a
+logger and walk through an example logger setup. 
+
+Background
+~~~~~~~~~~
+
+The MongoDB Java driver uses the Simple Logging Facade For Java (SLF4J).
+SLF4J allows you to specify your logging framework of choice at deployment time.
+For more information on SLF4J,
+`see the SLF4J documentation <http://www.slf4j.org/>`__.
+
+To set up a logger you must add the ``slf4j-api`` package, a logging framework, and a
+**binding** to your project's dependency list. A binding is a piece of code that
+connects the ``slf4j-api`` package with a logging framework. The example below shows
+how to bind the ``slfj-api`` package to the two most popular logging frameworks, Log4j2 and
```

Comment:
typo?
```suggestion
how to bind the ``slf4j-api`` package to the two most popular logging frameworks, Log4j2 and
```

Reply:
Thanks for the note. Good catch, apologies for letting that slip through. Fix: https://github.com/mongodb/docs-java/pull/76/commits/b05d369d84c141f18d77e64e8c5c905239a02c03